crates/ox_content_ssg/src/plugins/social-tweet-full-media.css

```
@@ -193,6 +193,18 @@
   display: none;
 }
 
+.ox-tweet--full .ox-tweet__copy-status {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)